packages/components/buttons/primary-button/src/primary-button.styles.ts

```diff
@@ -34,7 +34,6 @@ const getButtonStyles = (
     align-items: center;
     color: ${designTokens.colorSurface};
     transition: background-color ${designTokens.transitionLinear80Ms};
-    font-size: ${designTokens.fontSizeDefault};
```

Comment:
This font size was the same as the one in `AccessibleButton` (which this component relies on).
UX folks told me all buttons must have the same font size so I think it's better to only have the value set over there.